src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,11 @@ public class CliSyntax {
     // TODO: Change and remove this
     public static final Prefix PREFIX_ADDRESS = new Prefix("t/");
     public static final Prefix PREFIX_ALIAS = new Prefix("a/");
+    public static final Prefix PREFIX_STUDENT_ID = new Prefix("sid/");
+    public static final Prefix PREFIX_DATETIME = new Prefix("d/");
+    public static final Prefix PREFIX_ATTEND = new Prefix("at/");
```

Comment:
```suggestion
    public static final Prefix PREFIX_ATTEND = new Prefix("att/");
```

So that it makes more sense that ti refers to attendance.